ledger/store/src/transaction/deployment.rs

```diff
@@ -184,9 +195,10 @@ pub trait DeploymentStorage<N: Network>: Clone + Send + Sync {
         let program_id = *program.id();
 
         atomic_batch_scope!(self, {
-            // Store the program ID.
-            self.id_map().insert(*transaction_id, program_id)?;
-            // Store the edition.
+            // Store the program ID and edition.
+            // Note that the `id_map_v1` is intentionally excluded.
```

Comment:
Despite there being follow up PRs, might be nice to add a similarly explicit note about id_map_v1 being deprecated in favour of id_map_v2.